ui/public/locales/pt_BR.json

```diff
@@ -1,877 +1,1143 @@
+"label.action.iso.permission": "Atualizar permiss\u00f5es da ISO",
+"label.action.iso.share": "Atualizar compartilhamento da ISO",
 "label.action.list.nexusvswitch": "Listar NexusVswitch",
 "label.action.lock.account": "Bloquear conta",
 "label.action.lock.account.processing": "Bloqueando conta....",
-"label.action.manage.cluster": "Vincular Cluster",
-"label.action.manage.cluster.processing": "Vinculando o Cluster....",
+"label.action.manage.cluster": "Vincular cluster",
+"label.action.manage.cluster.processing": "Vinculando o cluster....",
 "label.action.migrate.instance": "Migrar Inst\u00e2ncia",
 "label.action.migrate.instance.processing": "Migrando Inst\u00e2ncia....",
-"label.action.migrate.router": "Migrar Roteador",
-"label.action.migrate.router.processing": "Migrando Roteador...",
+"label.action.migrate.router": "Migrar roteador",
+"label.action.migrate.router.processing": "Migrando roteador...",
+"label.action.migrate.router.to.ps": "Migrar roteador para outro Armazenamento Prim\u00e1rio",
 "label.action.migrate.systemvm": "Migrar VM de Sistema",
 "label.action.migrate.systemvm.processing": "Migrando VM de Sistema...",
+"label.action.migrate.systemvm.to.ps": "Migrar VM de Sistema para outro Armazenamento Prim\u00e1rio",
 "label.action.project.add.account": "Adicionar conta ao projeto",
+"label.action.project.add.user": "Adicionar usu\u00e1rio a um projeto",
 "label.action.reboot.instance": "Reiniciar Inst\u00e2ncia",
 "label.action.reboot.instance.processing": "Reiniciando Inst\u00e2ncia...",
-"label.action.reboot.router": "Reiniciar Roteador",
-"label.action.reboot.router.processing": "Reiniciando Roteador....",
+"label.action.reboot.router": "Reiniciar roteador",
+"label.action.reboot.router.processing": "Reiniciando roteador....",
 "label.action.reboot.systemvm": "Reiniciar VM de Sistema",
 "label.action.reboot.systemvm.processing": "Reiniciando VM de Sistema....",
-"label.action.recover.volume": "Recover Volume",
+"label.action.recover.volume": "Recuperar volume",
 "label.action.recurring.snapshot": "Snapshots recorrentes",
 "label.action.register.iso": "Registrar ISO",
-"label.action.register.template": "Registrar Template da URL",
+"label.action.register.ncc": "Registrar NCC",
+"label.action.register.template": "Registrar template a partir da URL",
 "label.action.release.ip": "Liberar IP",
 "label.action.release.ip.processing": "Liberando IP....",
-"label.action.remove.host": "Remover Host",
-"label.action.remove.host.processing": "Removendo Host....",
-"label.action.reset.password": "Recuperar Senha",
-"label.action.reset.password.processing": "Recuperando a Senha....",
-"label.action.resize.volume": "Resize Volume",
-"label.action.resize.volume.processing": "Resizing Volume....",
-"label.action.resource.limits": "Limite de Recursos",
+"label.action.remove.host": "Remover host",
+"label.action.remove.host.processing": "Removendo host....",
+"label.action.remove.vm": "Liberar VM",
```

Comment:
Should be `"label.action.remove.vm": "Remover VM",` or `"label.action.release.vm": "Liberar VM",`?